imap-to-local-html.py

```diff
@@ -335,6 +335,29 @@ def getMailFolders():
             if mailFolders[menu]["parent"] == menusWithNoParent[0]:
                 mailFolders[menu]["parent"] = ""
 
+    # Add in any additional folders that may be mbox only
+    for i in server.get('folders'):
```

Comment:
So `i` is the folder name, right? Maybe a better name would be great

Reply:
Updated my main to have better naming